### ISSUE TITLE
Parsing edges wrong with useConnections

### ISSUE THREAD

**Issue by EloB** [monojack/graphql-normalizr#11]
See example below:

https://codesandbox.io/s/74vzqjw2j0

```js
const { normalize } = new GraphQLNormalizr({
  useConnections: true
});

const response = {
  data: {
    view: {
      __typename: "View",
      id: "2",
      viewables: {
        __typename: "ViewableConnection",
        edges: [
          {
            __typename: "ViewableEdge",
            node: {
              __typename: "Movie",
              id: "6"
            }
          },
          {
            __typename: "ViewableEdge",
            node: {
              __typename: "Show",
              id: "7"
            }
          }
        ]
      }
    }
  }
};

const result = normalize(response);
```

# Outputs
```js
{
  "views": {
    "2": {
      "id": "2",
      "viewables": [
        "6",
        "7"
      ]
    }
  },
  // What happends here?
  "viewableEdges": {
    "undefined": {
      "node": "7"
    }
  },
  "movies": {
    "6": {
      "id": "6"
    }
  },
  "shows": {
    "7": {
      "id": "7"
    }
  }
}
```

**Comment by EloB** [comment]
Related: https://github.com/monojack/graphql-normalizr/pull/13

**Comment by monojack** [comment]
fixed in b836db353210c220d7577905bae6ec2c5a78ca0f
